e2e-tests/tfm/cypress/e2e/journeys/deal-cancellation/reason for-cancelling.spec.js

```diff
@@ -0,0 +1,77 @@
+import relative from '../../relativeURL';
+import MOCK_DEAL_AIN from '../../../fixtures/deal-AIN';
+import { ADMIN, BANK1_MAKER1, PIM_USER_1 } from '../../../../../e2e-fixtures';
+import caseDealPage from '../../pages/caseDealPage';
+import reasonForCancellingPage from '../../pages/deal-cancellation/reason-for-cancelling';
+import { backLink, cancelLink, continueButton, errorSummary } from '../../partials';
+
+context('Amendments underwriting - add banks decision - proceed', () => {
```

Comment:
Seems misaligned with the file name (reason for cancelling)?

Also can missing a `-` in the file name 😛 

```
reason for-cancelling.spec.js => reason-for-cancelling.spec.js
```

Reply:
oops!